src/tsickle.ts

```diff
@@ -50,8 +51,43 @@ export interface AnnotatorOptions {
 
 export enum AnnotatorFeatures {
   LowerDecorators = 1 << 0,
+  /**
+   * filter out types when expanding `export * from ...`
+   */
+  FilterTypesInExportStart = 1 << 1,
+  /**
+   * emit enums as
```

Comment:
This is not temporary.

TSC works as this:
1. parse and type check / create symbol table
2. run transformers
3. emit

If we are expanding `*` into concrete identifiers, TypeScript has no information about them, and therefore cannot elide types from them. So we have to do this.
In the version of tsickle where we create a new pogram, TypeScript will recreate the symbol table and therefore know which types to elide.

So for transformers, we always want to do this.
Without transformers, we must not do this, as we rely on the fact that we produce functions for interfaces, which should not be elided.